2-customization/README.rst

```diff
@@ -53,14 +52,13 @@ Customize
 If you go to ``http://192.168.50.10/``, you will see the default Invenio,
 but how we can customize it? Let's first stop invenio server.
 
-Open with your favorite editor
-``/vagrant/2-customization/invenio-beauty/invenio_beauty/config.py``
+Open with your favorite editor the ``~/.virtualenvs/invenio/var/instance/invenio.cfg``
 
 1. Modify the logo
 ^^^^^^^^^^^^^^^^^^
 Let's make our theme beautiful by replacing the logo
 
-in the ``/vagrant/2-customization/invenio-beauty/invenio_beauty/config.py``
+in the ``~/.virtualenvs/invenio/var/instance/invenio.cfg`` add the following:
```

Comment:
hmm, then we need each time to explain them ``cdvirtualenv var/instance/`` and then ``vi invenio.cfg`` :/